services/ResourceService/composer.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "islandora/resource-service",
+    "description": "RESTful service providing resources in Fedora 4",
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "/Users/dpino/Desktop/Development/ISLANDORAWORK/CLAW_MICRO/chullo"
```

Comment:
completely. Not needed anymore, since we have agreement on chullo. Will change this during the day. Thanks @whikloj